### DISCUSSION TITLE
Unable to register

### DISCUSSION THREAD

**Discussion by tie61** [vercel/community#3600] [Account Issue] [answered]
### Question

Too many attempts. Triggered risk control cannot be registered, please cancel.   datie19960601@gmail.com

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
